playwright-e2e/tests/brain/brain-regression-test.spec.ts

```diff
@@ -0,0 +1,386 @@
+    await page.getByRole('heading', { name: 'What if my child or I have questions?' }).click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.getByRole('heading', { name: 'Consent & Assent Form' }).click();
+    await page.getByRole('heading', { name: 'Introduction' }).click();
+    await page.getByText('You are being invited to enroll your child in a research study that will collect').click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.getByRole('heading', { name: 'Consent & Assent Form' }).click();
+    await page.getByRole('heading', { name: 'Documentation of Consent' }).click();
+    await page.locator('#mat-radio-2').getByText('Yes').click();
+    await page.locator('#mat-radio-5').getByText('Yes').click();
+    await page.locator('#mat-input-0').click();
+    await page.locator('#mat-input-0').fill('Kid');
+    await page.locator('#mat-input-0').press('Tab');
+    await page.locator('#mat-input-1').fill('Zimmer');
+    await page.getByLabel('MM').click();
+    await page.getByLabel('MM').fill('01');
+    await page.getByLabel('DD').fill('01');
+    await page.getByLabel('YYYY').fill('2013');
+    await page.locator('#mat-input-5').fill('A');
+    await page.locator('#mat-input-5').click();
+    await page.locator('#mat-input-5').fill('Andrew');
+    await page.locator('div:nth-child(9) > .ddp-single-question > div > ddp-activity-question > .ddp-activity-question > ddp-activity-answer > ddp-activity-text-answer > ddp-activity-text-input > .activity-text-input > .mat-form-field > .mat-form-field-wrapper > .mat-form-field-flex > .mat-form-field-infix').click();
+    await page.locator('#mat-input-6').fill('Zimmer');
+    await page.getByLabel('Full Name').click();
+    await page.getByRole('combobox', { name: 'Full Name' }).fill('Andrew Zimmer');
+    await page.getByText('Parent', { exact: true }).click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.getByRole('heading', { name: 'Consent & Assent Form' }).click();
+    await page.getByRole('heading', { name: 'Research Assent Form – Child/Adolescent' }).click();
+    await page.getByText('We want to tell you about a research study we are doing. A research study is a w').click();
+    await page.locator('div').filter({ hasText: 'Full Name' }).nth(2).click();
+    await page.getByRole('combobox', { name: 'Full Name' }).fill('Kid Zimmer');
+    await page.getByRole('button', { name: 'Submit' }).click();
+    await page.getByRole('heading', { name: 'Medical Release Form' }).click();
+    await page.getByText('Thank you very much for your consent to have your child participate in this rese').click();
+    await page.getByLabel('Full Name *').click();
+    await page.getByLabel('Full Name *').fill('KID ZIMMEr');
+    await page.getByLabel('Full Name *').press('Tab');
+    await page.getByRole('combobox', { name: 'Country/Territory Country/Territory' }).getByText('Country/Territory').click();
```

Comment:
@aweng98 @Rinchan20 this test is much more copy-pasta from playwright generator.  The osteo test does a better job of taking what was code like this and making better use of page classes and existing utilities.  For RGP stuff I'm thinking we could do something similar--first draft the generator code, make it pass, and then refactor the tests into page classes and make use of shared utilities.  How's that sound for a general approach to playwright going foward?

Reply:
just saw this and yeah - sounds like a plan - disregard above comment